tests/src/test/java/alluxio/client/fuse/JNIFuseIntegrationTest.java

```diff
@@ -45,10 +43,4 @@ public void mountFuse(FileSystem fileSystem, String mountPoint, String alluxioRo
   public void umountFuse(String mountPath) throws Exception {
     mFuseFileSystem.umount();
   }
-
-  @Ignore
-  @Override
-  public void touchAndLs() throws Exception {
-    // TODO(lu) Enable the test after https://github.com/Alluxio/alluxio/issues/13090 solved
-  }
```

Comment:
shall we apply the same to `alluxio.server.worker.WorkerFuseIntegrationTest#touchAndLs` ?

Reply:
good catch!